tests/func_test.py

```diff
@@ -31,6 +33,36 @@ def delete(self):
         # Reference db_conn to test for AttributeError
         self.db_conn
 
+meta_schema = Draft4Validator.META_SCHEMA.copy()
+meta_schema['definitions']["simpleTypes"]['enum'].append('int')
+
+default_types = Draft4Validator.DEFAULT_TYPES.copy()
+default_types['int'] = int
+
+
+ExtendedDraft4Validator = \
+    create(meta_schema,
```

Comment:
Maybe move the `create(` up and then have the arguments on individual lines. Looks cleaner that way.